roofit/roofitcore/CMakeLists.txt

```diff
@@ -476,9 +476,9 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RooFitCore
   DICTIONARY_OPTIONS
     "-writeEmptyRootPCM"
   LIBRARIES
+    RooBatchCompute
```

Comment:
Doesn't think link `RooBatchCompute`? I thought the purpose was to `dlopen` it as needed?